libs/EXTERNAL/libtatum/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.16)
 
 project("tatum")
 
-set(TATUM_EXECUTION_ENGINE "auto" CACHE STRING "Specify the framework for (potential) parallel execution")
+set(TATUM_EXECUTION_ENGINE "serial" CACHE STRING "Specify the framework for (potential) parallel execution")
```

Comment:
Should comment this -- default set to serial for now as we're getting nondeterministic results from Tatum with TBB.